test/lit/passes/optimize-casts.wast

```diff
@@ -30,8 +35,8 @@
   ;; CHECK-NEXT:  )
   ;; CHECK-NEXT: )
   (func $ref.as (param $x (ref null $A))
-    ;; After the first ref.as, we can use the cast value in later gets, which is
-    ;; more refined.
+    ;; We duplicate the ref.as to the first local.get, since it is more refined
+    ;; than the local.get alone. We then use the refined index throughout.
     (drop
       (local.get $x)
     )
```

Comment:
The comment on line 51 now applies to line 44 as well: on both of them we don't need the RefAs any more, but we leave it there for other opts.

Reply:
Done.